### PR TITLE
Added oms-mailer

### PR DESCRIPTION
**Pull request by blacksph3re**

### CODE REVIEW

**Comment by serge1peshcoff** [comment]
I will approve, but after postfix integration

**Review by serge1peshcoff — APPROVED** [review]
okay I approve :D

**Comment by blacksph3re** [comment]
OK seems I can not merge because I don't have write access.

**Comment by serge1peshcoff** [comment]
I will then

**Pull request merged**
